examples/misc/rc_traits_sgskip.py

```diff
@@ -107,7 +107,7 @@ def float_to_rgba(ob, name, val):
 
 
 def file_exists(ob, name, val):
-    fh = file(val, 'r')
+    fh = open(val, 'r')
```

Comment:
These questions are more directed at the mpl devs then @cclauss:

if this is just checking if the file exists, why can't it be replaced with "os.path.exists"?
and I know this won't get changed since it'd be an API change, but what's the point of obj and name if they're not used in the function?

Reply:
Even better, that function isn't actually used.